chain/cosmos/chain_node.go

```diff
@@ -625,7 +625,7 @@ func (tn *ChainNode) CreateNodeContainer() error {
 }
 
 func (tn *ChainNode) StopContainer() error {
-	return tn.Pool.Client.StopContainer(tn.Container.ID, uint(time.Second*30))
+	return tn.Pool.Client.StopContainer(tn.Container.ID, 30)
```

Comment:
Unit of time is already seconds. This 3rd party package made an interesting choice. For time args, I almost always use `time.Duration`.

Reply:
Great catch. We should fix this at the upstream too. https://github.com/strangelove-ventures/horcrux/pull/76